soap/soap_connector.bal

```diff
@@ -32,10 +32,26 @@ public type SoapConnector object {
         P{{request}} Request to be sent
         R{{}} If success, returns the response object, else returns `SoapError` object
     }
-    public function sendSoapRequest(string path, SoapRequest request) returns (SoapResponse|SoapError);
+    public function sendReceive(string path, SoapRequest request) returns SoapResponse|SoapError;
+
+    documentation {
+        Send Robust requests.Sends the request and possibly receives an error.
+        P{{path}} Resource path
+        P{{request}} Request to be sent
+        R{{}} If success, returns nothing, else returns `SoapError` object
```

Comment:
Can we say returns nil instead of "nothing"?

Reply:
Fixed